tello Test/tello_web_server.py

```
@@ -799,6 +1528,283 @@ async def rotate_handler(request: web.Request) -> web.Response:
+        # 3. 前進移動
+        import re
+        distance_match = re.search(r'(\d+)\s*cm', message)
+        distance = int(distance_match.group(1)) if distance_match else 50
+        
+        logger.info(f"Tello前進移動コマンドを実行中... 距離: {distance}cm")
+        move_result = await tello_controller.move('forward', distance)
+        if move_result.get('success', False):
+            responses.append(f"✅ 前に{distance}cm移動しました。")
+            await asyncio.sleep(2)  # 移動安定化のため待機
+        else:
+            responses.append(f"❌ 前への移動に失敗しました。")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)